syntax = "proto3";

package foreverbull.backtest;

option go_package = "github.com/lhjnilsson/foreverbull/pkg/backtest/pb";

import "foreverbull/common.proto";

enum IngestionStatus {
    CREATED = 0;
    DOWNLOADING = 1;
    INGESTING = 2;
    COMPLETED = 3;
    ERROR = 4;
}

message Ingestion {
    foreverbull.common.Date start_date = 1;
    foreverbull.common.Date end_date = 2;
    repeated string symbols = 3;
}
